blog/2024-10-26/if-kakaoai-2024-report.md

```
@@ -0,0 +1,583 @@
+## 패널톡 1부
+
+각 계열사 CTO 들이 나와서 이야기하고 질의응답
+
+> AI Finance
+
+### 카카오뱅크
+
+> AI Ecosystem
+
+- 카카오뱅크는 다양한 AI 기술을 사용하여 성과를 내고 국제적인 인정을 받고 있음
+    - 고객 상담의 1/3 이상을 AI 를 사용하여 처리 중
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)